src/main/scala/com/agar/core/Agar.scala

```diff
@@ -2,26 +2,44 @@ package com.agar.core
 
 
 import akka.actor.{ActorRef, ActorSystem}
-import com.agar.core.context.{AgarSystem, DefaultAgarSystem}
 import com.agar.core.arbritrator.Arbitrator
-import com.agar.core.arbritrator.Player.StartGameTurn
+import com.agar.core.arbritrator.Protocol.StartGameTurn
+import com.agar.core.context.AgarSystem
 import com.agar.core.logger.Journal
+import com.agar.core.region.Protocol.InitRegion
 import com.agar.core.region.Region
-import com.agar.core.region.Region.InitRegion
+import com.typesafe.config.ConfigFactory
 
-//#main-class
+import scala.concurrent.duration._
 
 object Agar extends App {
 
-  implicit val context: AgarSystem = DefaultAgarSystem
+  val nbPlayer = getArg(0).getOrElse("1").toInt
```

Comment:
We should put `2` instead of one.

Reply:
Agree otherwise the game is stupid. In addition, all material dedicated to clustering has been removed (this is in another PR)